gateway-server/src/main/java/org/apache/knox/gateway/config/impl/GatewayConfigImpl.java

```diff
@@ -233,6 +233,12 @@
   static final String REMOTE_ALIAS_SERVICE_CONFIG_PREFIX = GATEWAY_CONFIG_FILE_PREFIX + ".remote.alias.service.config.prefix";
   static final String REMOTE_ALIAS_SERVICE_CONFIG_PREFIX_DEFAULT = GATEWAY_CONFIG_FILE_PREFIX + ".remote.alias.service.config.";
 
+  public static final String REFRESHABLE_SERVIVCE_PARAMETERS_FILENAME = "refreshable-service-parameters.xml";
+  private static final String REFRESHABLE_SERVICE_PARAMETERS_FOLDER = GATEWAY_CONFIG_FILE_PREFIX + ".refreshable.service.parameters.folder";
+  private static final String REFRESHABLE_SERVICE_PARAMETERS_FOLDER_MONITORING_INTERVAL = GATEWAY_CONFIG_FILE_PREFIX
+          + ".refreshable.service.parameters.folder.monitor.interval";
+  private static final int REFRESHABLE_SERVICE_PARAMETERS_FOLDER_MONITORING_INTERVAL_DEFAULT = 60000;
```

Comment:
I thought an existing file monitor is being re-used, but this implies a new monitor.